package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qsc-workspace",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```

Comment:
What node version are you on? Interesting that it's gone back a version here and removed a bunch of fields from each dependency record.